pithos/pithos.py

```diff
@@ -1460,22 +1463,63 @@ def restore_position(self):
         if not x is None and not y is None:
             self.move(int(x), int(y))
 
+    def restore_size(self):
+        """ resizes window to size stored in preferences """
+        w, h = self.settings['win-size']
+        if w is not None and h is not None:
```

Comment:
These can never be `None`.

Reply:
I was just following what you did with the position.

Reply:
Past me was an idiot =)